tools/noise/subr.sh

```diff
@@ -299,6 +299,9 @@ get_image_iso() {
 	if [ -z "$iso" -o "$iso" = "65535" ]; then
 		iso=$(get_exif_key "$file" Exif.Photo.StandardOutputSensitivity)
 	fi
+	if [ -z "$iso" -o "$iso" = "65535" ]; then
```

Comment:
I thought about it. I did not find any information which tag should be created  on http://www.exiv2.org/tags.html 
I'd say keep it and if there are same images which only write these tags we can add them.